docs/modules/operation/pages/quick-start/notification-config.adoc

```diff
@@ -0,0 +1,63 @@
+
+= Step 8: Configure Notifications
+
+This section describes how to configure and customize your {page-component-title} notifications.
+
+NOTE: This step is separate from xref:operation:quick-start/notifications.adoc[setting up default notifications].
+We assume that you have followed the previous steps and are ready to proceed to more in-depth work with notifications.
+
+== Configuration and customization
+
+Now that you have xref:operation:quick-start/baseline.adoc[established your environment's baseline], it's time to take your notes and configure your custom notifications.
+Establishing notifications that are tailored for your environment and monitoring requirements lets the right people see and respond to the right issues in a timely manner.
+
+Here, we will describe how to configure event notifications and path outages.
+Event notifications can be configured to provide UEI-specific coverage, allowing for granular control of incoming information.
+Path outages prevent the generation of excess notifications by identifying network paths where failures may cause multiple nodes to appear offline.
+
+== Configure event notifications
+
+An event notification defines a destination path and message template for a specific event UEI.
+It is possible to define multiple notifications for the same UEI to use different filters, destination paths, and message templates.
+Follow these steps to configure event notifications:
+
+. Click the *Gear* symbol in the top-right of the screen.
+. Under Event Management, click menu:Configure Notifications[Configure Event Notifications].
+A list of events that are configured for notifications is displayed.
+. Click *Add New Event Notification*, choose the UEI that will trigger a notification, and click *Next*.
+** (Optional) Build a rule to match a subset of IP addresses or services, and click menu:Validate Rule Results[Next] or *Skip Results Validation*.
```

Comment:
I think this needs to be called out as a step... while it is optional, the flow has the user going to the "Build Rule" page and selecting those IPs and Services.  I feel like this should be a step like the others but noted as being optional and how the rule will apply if they opt not to change anything.

Reply:
I personally tend to like moving optional steps into a sub-step rather than a numbered one, but it's definitely not a standard thing; I'll discuss it with Bonnie when she gets back, and change this step to a proper numbered one for now.

As for how the rule will apply should the user not make those optional changes, I'll need to do a quick bit of research and then update. :)